docs/tutorials/deploy-grafana-dashboard.md

```diff
@@ -1,28 +1,38 @@
-# Deploy Grafana Dashboard
+# Access Tracee Metrics in Prometheus and Grafana
```

Comment:
can you maybe add a sentence about why someone might be interested in this? the they can monitor the health and operation of tracee and troubleshoot, like any other cloud native app, using prometheus metrics.

Reply:
Yes, good point